Sources/GoogleMapsUtils/Clustering/Algorithms/GridBasedCluster/GMUGridBasedClusterAlgorithm1.swift

```diff
@@ -58,19 +58,22 @@ final class GMUGridBasedClusterAlgorithm1: GMUClusterAlgorithm1 {
         var clusters: [Int : GMUCluster1] = [:]
 
         // Divide the whole map into a numCells x numCells grid and assign items to them.
-        let numCells: Double = Double(ceil(256 * pow(2, zoom) / Float(gmuGridCellSizePoints)))
+        let numCells: Int = Int(ceil(256 * pow(2, zoom) / gmuGridCellSizePoints))
```

Comment:
[self-review] ceil accepts `FloatingPoint`, so converted `gmuGridCellSizePoints` to `Float` from `Int`.